lib/node_modules/@stdlib/math/base/complex/divide/test/test.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var cdiv = require( './../lib' );
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof cdiv, 'function', 'main export is a function' );
+	t.end();
+});
+
+function testCdiv( a, b, c, d, e, f, label ) {
```

Comment:
One other small goal was simplifying the fancy output so that the tests read like this:

```
  ab <= SMALL_THRESHOLD

    ✔ ( 1e-304 + 2e-304j ) / ( 1 + 2j ) == 1e-304 + 0j
```

and can be fed into a python (or similar) interpreter directly for cross-comparison.